mettle/src/mtwister.h

```diff
@@ -0,0 +1,18 @@
+#ifndef __MTWISTER_H
+#define __MTWISTER_H
+
+#include <stdint.h>
+
+#define STATE_VECTOR_LENGTH 624
+#define STATE_VECTOR_M      397 /* changes to STATE_VECTOR_LENGTH also require changes to this */
```

Comment:
Do we want a codification of this interdependency as a macro which resolves `M` from `LENGTH` or maybe a bit more elaboration of the relationship in the comment?

Reply:
This is the original code from the marsenne twister implementation (not done by me), not sure exactly what is the correlation between the two variables, probably is explained in the original paper.